### PR TITLE
Fix #2879:added build.bazel file in utility module

### LINKED ISSUE #2879: Create a BUILD.bazel file for the gcsresource package in the utility module
This task is part of the build system migration from Gradle to Bazel. 

## Prerequesites

- Make sure you are able to build Oppia using Bazel
- Follow the [Oppia Bazel Setup instructions](https://github.com/oppia/oppia-android/wiki/Oppia-Bazel-Setup-Instructions) to setup Bazel and learn the terminology.
- Read the [Gradle Bazel Migration Best Practies and FAQ](https://github.com/oppia/oppia-android/wiki/Gradle--Bazel-Migration-Best-Practices-and-FAQ) for more context and see if there's an answer there to any question you have.

## Task

1. Create a BUILD.bazel file in

`src/main/java/org/oppia/android/util/gcsresource/`

2. Rename GcsResouceAnnotations.kt to GcsResourceAnnotations.kt

3. Define the kt_android_library libraries

Based on the imports in the Kotlin files, add the required third_party dependencies to make the libraries build. Check the Useful links section for a list of third_party dependencies.

You can start with the following stub

```
load("@dagger//:workspace_defs.bzl", "dagger_rules")
load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")

kt_android_library(
    name = "annotations",
    srcs = [
        "GcsResourceAnnotations.kt",
    ],
    visibility = ["//:oppia_api_visibility"],
)

kt_android_library(
    name = "prod_module",
    srcs = [
        "GcsResourceModule.kt",
    ],
    deps = [
        ":dagger",
    ],
)

dagger_rules()
```

3. Add the directory to the MIGRATED_PROD_FILES of the module's root BUILD.bazel file

4. Add the libraries to the list of deps dependencies in the targets that require the library

5. Clean up any unused deps dependencies

6. Verify that the app builds

## Useful links

- [BUILD.bazel files](https://github.com/oppia/oppia-android/search?q=filename%3ABUILD.bazel&type=code)
- [Visibilities](https://github.com/oppia/oppia-android/blob/76dc13dba3ad77aecd2218a5372949d6551f12fe/BUILD.bazel)
- [third_party dependencies](https://github.com/oppia/oppia-android/blob/a70d9b016f0d77dd30c4939041f82ea02ee28d1d/third_party/versions.bzl)

### PR DESCRIPTION
**Pull request by Akshat-gour**
Fixes #2879:
1.Create a BUILD.bazel file in src/main/java/org/oppia/android/util/gcsresource/
2.Rename GcsResouceAnnotations.kt to GcsResourceAnnotations.kt
3.Added the kt_android_library libraries
4.Added the directory to the MIGRATED_PROD_FILES of the Utility BUILD.bazel file
5.Added the libraries to the list of deps dependencies in the targets that require the library

## Checklist

- [X] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [X] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [X] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [X] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [X] The PR is made from a branch that's **not** called "develop".
- [X] The PR is made from a branch that is up-to-date with "develop".
- [X] The PR's branch is based on "develop" and not on any other branch.
- [X] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by fsharpasharp** [comment]
Can you pull the latest version of the repository and rerun the tests?

**Comment by Akshat-gour** [comment]
@fsharpasharp i checked its the latest version

**Comment by fsharpasharp** [comment]
Try to make investigate why the tests are failing and why the app doesn't build. Is it the same issue as before?

Are you able to build the gcsresource package in isolation?

**Comment by Akshat-gour** [comment]
can you tell me the cmd used to build the gcsresource package in isolation

**Comment by fsharpasharp** [comment]
Easiest way is to cd to the gcsresource directory and running

`bazel build :all`

**Comment by Akshat-gour** [comment]
This error is occuring can you guide me
image

File: utility/src/main/java/org/oppia/android/util/gcsresource/GcsResouceAnnotations.kt [hunk 1]
```diff
@@ -1,10 +0,0 @@
-// ktlint-disable filename
-package org.oppia.android.util.gcsresource
-
-import javax.inject.Qualifier
-
-/** Corresponds to the default GCS resource bucket name. */
-@Qualifier annotation class DefaultResourceBucketName
-
-/** Corresponds to the GCS resource bucket name for questions. */
-@Qualifier annotation class QuestionResourceBucketName
```
Comment (FareesHussain) [inline utility/src/main/java/org/oppia/android/util/gcsresource/GcsResouceAnnotations.kt]: Why is this removed here?

**Review by FareesHussain — CHANGES_REQUESTED** [review]

File: utility/src/main/java/org/oppia/android/util/gcsresource/GcsResouceAnnotations.kt (hunk 1 above)
Comment (Akshat-gour) [reply utility/src/main/java/org/oppia/android/util/gcsresource/GcsResouceAnnotations.kt]: got removed by mistake added again
